content/millennium/r4/base/entities/location.md

```diff
@@ -40,15 +40,15 @@ The following fields are returned if valued:
 
```

Comment:
can't comment on the line since it wasn't changed, but let's update 'Location id' to 'Location ID'